assemblies/dynamic-plugins/assembly-third-party-plugins-installation.adoc

```diff
@@ -0,0 +1,27 @@
+[id="assembly-third-party-plugins-installation"]
+= Third-party plugins in {product}
+:context: assembly-third-party-plugins-installation
+
+In {product}, third-party dynamic plugins can be integrated seamlessly, enabling the use of external plugins. This flexibility allows you to enhance the platform functionality without modifying its source code and rebuilding. To add third-party dynamic plugins, you need to export them as derived package.
+
+The third-party plugins are typically packaged as dynamic plugin derived package, enabling you to manage external modules without rebuilding the entire application. The process involves exporting the plugin package, ensuring that dependencies are correctly bundled or marked as shared, depending on their relationship to the {product-short} environment.
```

Comment:
Consider removing repeated content. Note that this sentence is still a definition of the step in line 12.

```suggestion
While exporting the plugin package, you should ensure that dependencies are correctly bundled or marked as shared, depending on their relationship to the {product-short} environment.
```